### PR TITLE
Added more yield icons to the city stats table

### PR DESCRIPTION
**Pull request by xlenstra**
E.g.:
image
image
image

Alternatively, I could only put the culture and food icons after the second number, or use the happiness icon for an active 'we love the king' day.

### CODE REVIEW

**Comment by yairm210** [comment]
All the icons look unaligned to the text :/
I remember I had to manually change the small icons to raise them, did someone remake the icons from the originals or something?

**Comment by yairm210** [comment]
Also, please use Fonts.x instead of the character itself, so it's obvious what's going on

**Comment by SomeTroglodyte** [comment]
My vague memory says I re-did them once, but it also says there should be separate versions for font (50px) and larger (100? reduced from 200 when we had but one atlas spilling?) use, and I also vaguely remember tweaking the 50px ones for text baseline???

**Comment by yairm210** [comment]
My issues with the font shouldn't prevent this from being merged
But I'm not going to merge this myself to see if the '2 other collaberators merge PRs' system works :)

**Review by yairm210 — APPROVED** [review]

File: core/src/com/unciv/ui/cityscreen/CityStatsTable.kt [hunk 1]
```diff
@@ -74,17 +74,26 @@ class CityStatsTable(val cityScreen: CityScreen): Table() {
...
-            innerTable.add("Demanding [${cityInfo.demandedResource}]".toLabel()).row()
+        
+        val tableWithIcons = Table()
+        tableWithIcons.defaults().pad(2f)
+        if (cityInfo.isInResistance()) {
+            tableWithIcons.add(ImageGetter.getImage("StatIcons/Resistance")).size(20f)
+            tableWithIcons.add("In resistance for another [${cityInfo.getFlag(CityFlags.Resistance)}] turns".toLabel()).row()
+        }
+        if (cityInfo.isWeLoveTheKingDay()) {
+            tableWithIcons.add(ImageGetter.getStatIcon("Food")).size(20f)
```
Comment (will-ca) [inline core/src/com/unciv/ui/cityscreen/CityStatsTable.kt:90]: Nitpicky/highly subjective: Maybe the `Population` stat icon?

**Review by will-ca — APPROVED** [review]

File: core/src/com/unciv/ui/cityscreen/CityStatsTable.kt (hunk 1 above)
Comment (xlenstra) [reply core/src/com/unciv/ui/cityscreen/CityStatsTable.kt:90]: Was my first idea as well, but we don't have that as an emoji yet

Comment (xlenstra) [reply core/src/com/unciv/ui/cityscreen/CityStatsTable.kt:90]: To me food seems more logical, as the main benefit is the +20% Growth (surplus food) it gives, but an argument could be made for both

**Comment by SomeTroglodyte** [comment]
According to my mails there was mention that 'Population' is not yet there as emoji/font glyph.
- I did push the redone gimps to the repo, they're in [ImagesToNotAddToGame](https://github.com/yairm210/Unciv/tree/master/android/ImagesToNotAddToGame)
- To make a character from that, I take the full xcf, do a scale to 42px (NoHalo), paste as layer over the happiness 50px png, nudge the thing up using layer transparency till they align, throw away the helpers and export... Dunno whether github preserves png bits unchanged here, but you could try:
Population

**Pull request merged**